account_payment_order/models/bank_payment_line.py

```diff
@@ -58,6 +58,17 @@ class BankPaymentLine(models.Model):
         'res.currency',
         related='order_id.payment_mode_id.company_id.currency_id',
         readonly=True, store=True)
+    purpose_code = fields.Selection(selection=[
```

Comment:
Instead of repeating the same selection, use the same trick as we do in sale_order_type:

https://github.com/OCA/sale-workflow/blob/e1149e3f39de09cf1ce5094f9ba1845d3e43eadd/sale_order_type/models/sale_order_type.py#L15-L18